src/pages/trade/ui/chart.tsx

```diff
@@ -21,86 +22,88 @@ const ChartLoadingState = ({ height }: ChartProps) => {
   );
 };
 
-const ChartData = ({
-  height,
-  baseAsset,
-  quoteAsset,
-}: {
-  height: number;
-  baseAsset: Metadata;
-  quoteAsset: Metadata;
-}) => {
-  const chartElRef = useRef<HTMLInputElement>(null);
-  const chartRef = useRef<IChartApi | null>(null);
+const ChartData = observer(
```

Comment:
Using hook that observes mobx connection state store. Hennce the wraps for two components with `observer`.